sw/host/opentitanlib/src/test_utils/init.rs

```diff
@@ -48,8 +48,12 @@ pub struct InitializeTest {
 impl InitializeTest {
     pub fn init_logging(&self) {
         let level = self.logging;
+        // The tests might use OpenOCD which uses util::printer so we will get
+        // more useful logging if we log the target instead of the module path
         if level != LevelFilter::Off {
             env_logger::Builder::from_default_env()
+                .format_target(true)
```

Comment:
Please add this initialization change to opentitantool as well.

Reply:
done